src/main/scala/files_metadata/application/FilesMetaUseCases.scala

```diff
@@ -228,4 +228,31 @@ class FilesMetaUseCases {
 
     repository.updateFileParent( fileUUID, newParentUUID )
   }
+
+  def unShareFile(
+      ownerUUID: UUID,
+      fileUUID: UUID,
+      otherUserUUID: UUID
+  ): Unit = {
+    val fileMeta = repository.getFileMeta( fileUUID )
+
+    if (fileMeta.ownerUuid != ownerUUID) {
+      throw DomainExceptions.FileNotOwnedException(
+        "The user does not own the file"
```

Comment:
Try to write errors than can be shown in the UI. Instead of "The user does not own the file" write "You don't own the file".

Reply:
Repaired: 
![image](https://github.com/hawks-atlanta/metadata-scala/assets/47455237/6445e4a1-15b4-4e0d-81b3-d0c6328d2347)